api/src/index.ts

```diff
@@ -6,33 +6,30 @@ import { Color } from './node/color'
 import constants from './node/constants'
 import './node/exceptions'
 import { Player } from './node/player'
+import server from './node/server'
 
 declare global {
     namespace NodeJS {
         interface WriteStream {
             _orig_write: NodeJS.WriteStream['write'];
         }
     }
-
-    // var metacall: Function;
 }
 
-// global.metacall = (name: string, ...args: any[]) => {
-//     // console.log(`[metacall] ${name}(${args.join(', ')})`)
-// }
-
 export {
     events,
     chat,
     Color,
     constants,
-    Player
+    Player,
+    server
 }
 
 export default {
```

Comment:
Export things individually instead of 
export default Object

Reply:
There is 2 different exports